.copier-answers.yml

```diff
@@ -15,8 +16,10 @@ odoo_version: 16.0
 org_name: Odoo Community Association (OCA)
 org_slug: qrtl
 rebel_module_groups: []
-repo_description: ''
-repo_name: ''
+repo_description: OCA module
```

Comment:
@liuhehe1995 Why is this done differently to what you have done in https://github.com/qrtl/stx-custom/pull/2/files? We should stick to the same practice unless there is a good reason to do otherwise.

Reply:
Thank you for bringing this to my attention. 
I've looked up the information and for CI tests across all repositories, decided to utilize both Odoo and OCB flavors, ensuring a more comprehensive and uniform testing workflow.